File: Topic Based Problems/01 Fundamentals/1 Beginner/20 Inches to Centimeters/C++/InchestoCentimeters.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
    double inches;

    cout << "Enter the length in Inches: ";
    cin >> inches;

    cout << fixed << setprecision(2) << "Length in Centimeters = " << inches * 2.54 << endl;

    return 0;
}

File: Topic Based Problems/01 Fundamentals/1 Beginner/20 Inches to Centimeters/Java/InchestoCentimeters.java
import java.util.Scanner;

public class InchestoCentimeters {
    public static void main(String[] args) {
        double inches;

        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the length in Inches: ");
        inches = scanner.nextDouble();

        System.out.printf("Length in Centimeters = %.2f%n", inches * 2.54);

        scanner.close();
    }
}
